mesheryctl/internal/cli/root/system/reset.go

```diff
@@ -103,21 +103,9 @@ func resetMesheryConfig() error {
 
 	case "kubernetes":
 
-		version := currCtx.GetVersion()
-		if version == "latest" {
-			if currCtx.GetChannel() == "edge" {
-				version = "master"
-			} else {
-				version, err = utils.GetLatestStableReleaseTag()
-				if err != nil {
-					return err
-				}
-			}
-		}
-
 		log.Printf("Fetching Meshery Server and Meshery Operator manifests...")
 		// fetch the manifest files corresponding to the version specified
-		_, err := utils.FetchManifests(version)
+		_, err := utils.FetchManifests(currCtx)
```

Comment:
Could we make this as a struct function for context?

Reply:
@navendu-pottekkat from what I can see, `platform.go` imports `config`, so I don't think I can move `FetchManifests` to `config/config.go`, since it might result in import cycles.

Reply:
Fetching the version as a separate struct function? The `FetchManifests` can remain in `utils`.

Reply:
When I tried importing `utils` in `config/config.go`(for `GetLatestStableReleaseTag`), I faced an import cycle again since `utils` imports package `config`.

Reply:
If there's any other refactor you're considering, let me know and I can try to implement it in this PR itself.

Reply:
Okay, let's hold off this and merge this PR